domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/gtfs/transfers/Transfer.java

```diff
@@ -0,0 +1,178 @@
+         *
+         * @param toStopId a stop or station where a connection between routes ends
+         * @return builder for future object creation
+         */
+        public TransferBuilder toStopId(@NotNull final String toStopId) {
+            this.toStopId = toStopId;
+            return this;
+        }
+
+        /**
+         * Sets field transferType value and returns this
+         *
+         * @param transferType indicates the type of connection for the specified
+         * @return builder for future object creation
+         */
+        public TransferBuilder transferType(@NotNull final Integer transferType) {
+            this.transferType = TransferType.fromInt(transferType);
+            this.originalTransferTypeInteger = transferType;
+            return this;
+        }
+
+        /**
+         * Sets field minTransferTime value and returns this
+         *
+         * @param minTransferTime amount of time, in seconds, that must be available to permit a transfer between
+         *                        routes at the specified stops
+         * @return builder for future object creation
+         */
+        public TransferBuilder minTransferTime(@Nullable final Integer minTransferTime) {
+            this.minTransferTime = minTransferTime;
+            return this;
+        }
+
+        /**
+         * Returns an entity representing a row from transfers.txt if the requirements from the official GTFS
+         * specification are met. Otherwise, method returns an entity representing a list of notices.
+         *
+         * @return entity representing a row from transfers.txt if the requirements from the official GTFS specification
+         * are met. Otherwise, method returns an entity representing a list of notices.
+         */
+        public EntityBuildResult<?> build() {
+            noticeCollection.clear();
+
+            if (fromStopId == null || toStopId == null || transferType == null ||
+                    (minTransferTime != null && minTransferTime < 0)) {
+
+                if (fromStopId == null) {
+                    noticeCollection.add(new MissingRequiredValueNotice("transfers.txt",
+                            "from_stop_id", fromStopId + ";" + toStopId));
+                }
+                if (toStopId == null) {
+                    noticeCollection.add(new MissingRequiredValueNotice("transfers.txt",
+                            "to_stop_id", fromStopId + ";" + toStopId));
+                }
+                if (!TransferType.isEnumValueValid(originalTransferTypeInteger)) {
+                    noticeCollection.add(new UnexpectedEnumValueNotice("transfers.txt",
+                            "transfer_type", fromStopId + ";" + toStopId,
+                            originalTransferTypeInteger));
+                }
+                if (minTransferTime != null && minTransferTime < 1) {
+                    // here minTransferTime threshold should be user configurable
```

Comment:
Yes, let's do it!

Reply:
See https://github.com/MobilityData/gtfs-validator/issues/246